src/components/autocomplete/Autocomplete.js

```diff
@@ -57,17 +57,18 @@ const propTypes = {
   /**
    * Get the key for each item used when react creates the list.
    */
-  getItemKey: PropTypes.func.isRequired,
+  getItemKey: PropTypes.func,
```

Comment:
it was required because React complains when a key is not specified in a for loop.